### PR TITLE
Make "Found" status in GPX exports optional

### PR DESCRIPTION
**Pull request by auricgoldfinger**
When you export a cache or a list of caches to GPX, then all data is exported, including the "found" status of the cache, obviously indicating that you have found the cache.

However, when you want to export a cache or a list of caches so that a friend can import them (in any application) then this status is only confusing. It is not guaranteed that he found the same caches as well.

This change adds a checkbox to the "export to GPX" dialog to include/exclude the Found status. When checked (default), the sym tag of the exported GPX will contain the text "Geocache Found" for geocaches that were found. When unchecked, the sym tag of the exported GPX will always show "Geocache" and thus omits the word "Found". 

include_found_status

### CODE REVIEW

**Comment by Bananeweizen** [comment]
ok to test

File: main/res/values/preference_keys.xml [hunk 1]
```diff
@@ -128,6 +128,7 @@
     <string translatable="false" name="pref_settingsversion">settingsversion</string>
     <string translatable="false" name="pref_trackableaction">trackableaction</string>
     <string translatable="false" name="pref_shareafterexport">shareafterexport</string>
+    <string translatable="false" name="pref_includefoundstatus">includefoundstatus</string>
```
Comment (samueltardieu) [inline main/res/values/preference_keys.xml]: Do not include translations in your patch, those are handled through crowdin.net.

File: main/src/cgeo/geocaching/export/GpxExport.java [hunk 2]
```diff
@@ -96,11 +96,15 @@ private Dialog getExportDialog(final String[] geocodes, final Activity activity)
         final CheckBox shareOption = ButterKnife.findById(layout, R.id.share);
         shareOption.setChecked(Settings.getShareAfterExport());
 
+        final CheckBox includeFoundStatus = ButterKnife.findById(layout, R.id.include_found_status);
+        includeFoundStatus.setChecked(Settings.getIncludeFoundStatus());
```
Comment (samueltardieu) [inline main/src/cgeo/geocaching/export/GpxExport.java]: Note: check if this is needed or if the default state can be set in the layout itelf (same for "share").

File: main/res/values-nl/strings.xml [hunk 3]
```diff
@@ -558,6 +558,7 @@
   <string name="init_maptrail">Pad tonen</string>
   <string name="init_summary_maptrail">Laat pad op kaart zien</string>
   <string name="init_share_after_export">Open deel-menu na GPX export</string>
+  <string name="init_include_found_status">\"Gevonden\" status mee exporteren</string>
```
Comment (samueltardieu) [inline main/res/values-nl/strings.xml]: Do not include translations in your patch, those are handled through crowdin.net.

Comment (auricgoldfinger) [inline main/res/values-nl/strings.xml]: Ok, removed.

**Comment by samueltardieu** [comment]
Thanks for your contribution.

**Pull request merged**